src/Drupal/DrupalExtension/Context/BigPipeContext.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace Drupal\DrupalExtension\Context;
+
+use Drupal\big_pipe\Render\Placeholder\BigPipeStrategy;
+use Behat\Mink\Exception\UnsupportedDriverActionException;
+
+/**
+ * Big Pipe context.
+ */
+class BigPipeContext extends RawDrupalContext {
+
+    /**
+     * Prepares Big Pipe NOJS cookie if needed.
+     *
+     * @BeforeScenario
+     */
+    public function prepareBigPipeNoJsCookie()
+    {
+        try {
+            // Check if JavaScript can be executed by Driver.
+            $this->getSession()->getDriver()->executeScript('true');
```

Comment:
We were using a similar approach (https://github.com/drupaltest/behat-traits/blob/8.x-1.x/src/Traits/BrowserCapabilityDetectionTrait.php#L42) but this doesn't work with ChromeDriver which throws a fatal error when no page has been loaded yet (which is the case when running `@BeforeScenario`)